### DISCUSSION TITLE
Add custom cheatsheets ?

### DISCUSSION THREAD

**Discussion by ghost** [sudormrfbin/cheatsheet.nvim#18] [Q&A]
since not all extensions don't have cheatsheet, how could I add my own  ?

**Comment by sudormrfbin** [comment]
From the [Usage](https://github.com/sudormrfbin/cheatsheet.nvim/blob/master/README.md#usage) section:

> Your cheatsheet file is a simple text file with the name `cheatsheet.txt` found in `~/.config/nvim/` (`~/AppData/Local/nvim/` if you're on Windows) alongside your `init.vim`. Use the `:CheatsheetEdit` command to open it in a buffer to edit.

**Reply by molecula21** [reply]
Exactly what I was looking for. Thanks,

**Comment by Cruising0904** [comment]
This plugin is awesome! you are hero
